src/web/components/App.tsx

```diff
@@ -228,6 +229,10 @@ export const App = ({ CAPI, NAV }: Props) => {
         FocusStyleManager.onlyShowFocusOnTabs();
     }, []);
 
+    useEffect(() => {
```

Comment:
I would prefer we dynamically load this, it's not going to be used by our users so having it in the bundle doesn't make sense

Reply:
hmm, I cannot think of a way to do that without changing the bookmarklets in everyone's browsers.
Any ideas?
The bookmarklet just calls e.g. `javascript:window.guardian.readerRevenue.changeGeolocation()`

Reply:
The best I can think of is to update the bookmarklet scripts to e.g.:
```
window.guardian.initialiseReaderRevenue()
  .then(() => window.guardian.readerRevenue.changeGeolocation())
```

Reply:
To save updating bookmarks could you change them all in this code to roughly
```
window.guardian.readerRevenue.changeGeolocation = () => import(/* webpackChunkName: "readerRevenueHelpers" */ 'readerRevenueHelpers').then(({ default: _ }) => {
	changeGeoLocation()	
   }).catch(error => 'An error occurred while loading the reader revenue helpers');
```

Reply:
thanks for the tip @gtrufitt !
I've pushed a change which uses that strategy